testing/Overview.test.js

```diff
@@ -0,0 +1,109 @@
+    "shippingTo":["China"," Wales"," Belgium"," Japan"," France"," America"," Germany"," Canada"," Spain"," Brazil"," South Africa"," Belgium"," India"," Mexico"," Argentina"," Ethiopia"," Jamaica"," Brazil"," Sweden"," Denmark"," Switzerland"," Ghana"," Bangladesh"," Switzerland"," Nigeria"," Sri Lanka"," Ireland"," Italy"," Niger"," Sudan"," Italy"," Argentina"," Pakistan"," Cuba"," North Korea"],
+    "deliveryDate":["Oct","Oct"],
+    "_id":"5c6511ac8f191456a859ab5f","productId":98,"price":92.65,"description":"duis filet short bacon do aliquip nulla do hock in strip nulla in andouille t-bone ribeye ","options":{"ribs":["Picanha","est","meatloaf","andouille","Andouille","steak","quis","consequat","kevin"],"pig":["boudin","Magna","beef","hock","laborum","Excepteur","mignon","id","Ham","dolore","lorem"]},
+    "quantityInStock":37,
+    "availableToReturn":true,
+    "shippingCosts":2.99,
+    "manufacturingTime":4,
+    "shippingFrom":" France",
+    "__v":0
+  }
+]
+
+
+describe("Overview Component", () => {
+  
+  it('Should exist', () => {
+    const wrapper = shallow(<ItemDescription />);
+    expect(wrapper.exists()).toBe(true)
+  });
+  
+  it('Should render without throwing an error', () => {
+    expect(shallow(<ItemDescription />).find('.description').exists()).toBe(true);
+  });
+
+  it('Should render other modules when mounted', () => {
+    const props = {stateChange: true}
+    const wrapper = shallow(<ItemDescription {...props}/>);
+    // expect(wrapper.findWhere(node => node.is(SelectionList)).exists()).toBe(true)
+    expect(wrapper).toContain(<SelectionList />);
+  })
+  
+  it('Should call componentDidMount', () => {
+    const mount = jest.spyOn(ItemDescription.prototype, 'componentDidMount');
+    const wrapper = shallow(<ItemDescription />);
+    expect(ItemDescription.prototype.componentDidMount).toHaveBeenCalled();
+    mount.mockRestore();
+  });
+  
+  it('Should invoke functions after components mounted', () => {
+    const getFindQuantity = jest.spyOn(ItemDescription.prototype, 'findQuantity');
+    const getPostingDescription = jest.spyOn(ItemDescription.prototype, 'postingDescription')
+    mount(<ItemDescription />);
+    expect(getFindQuantity).toHaveBeenCalledTimes(1);
+    expect(getPostingDescription).toHaveBeenCalledTimes(1);
+  });
+  
+  it('Should stimulate a click event when adding to cart', () => {
+    const onButtonClick = sinon.spy(ItemDescription.prototype, 'addedToCart');
+    const wrapper = shallow(<ItemDescription addedToCart={onButtonClick}/>);
+    wrapper.find('.addToCart').simulate('click', { preventDefault() {} });
+    expect(onButtonClick.calledOnce).toBe(true)
+  });
+
+  it('Should open the modal when its state is opened', () => {
+    const props = {askedQuestion: true};
+    const wrapper = shallow(<ItemDescription {...props}/>);
+    wrapper.update();
+    expect(wrapper.find('.questionsModal').exists()).toEqual(true)
```

Comment:
same comment here, `find` returns `{}` if it fails to find.  so it will always exist